modules/gdscript/tests/scripts/parser/features/nested_arithmetic.out

```diff
@@ -76,7 +76,7 @@ GDTEST_OK
 
 -2.718
 
-36900.9009009009
+36900.9
```

Comment:
Seems like `num_real` hardcodes 6 "decimals" which in practice is 6 significant digits, so 36900.9.

`num` by default prints all the significant digits it can. IMO the `num_real` logic is needlessly restrictive.

Reply:
This shouldn't be blocking for this PR though but we need to look into it further, I think the hardcoded 6 doesn't make much sense especially since now `float` in GDScript in 64-bit.